contrib/cirrus/packer/fedora_setup.sh

```diff
@@ -19,9 +19,91 @@ req_env_var "
 
 install_ooe
 
+echo "Updating packages"
 ooe.sh sudo dnf update -y
 
-stub "Installing/configuring Fedora"
+ooe.sh sudo dnf -y module install cri-o:1.13
+
+echo "Installing dependencies"
+
+ooe.sh sudo dnf -y install \
+    atomic-registries \
+    btrfs-progs-devel \
+    buildah \
+    container-selinux \
+    containernetworking-plugins \
+    cri-o \
```

Comment:
Here too, I think this can just go away...unless it's being used to drag in other dependencies?